tests/Converter/Validator/Xsd2ValidatorTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+namespace GoetasWebservices\Xsd\XsdToPhp\Tests\Converter\Validator;
+
+class Xsd2ValidatorTest extends Xsd2ValidatorBase
+{
+
+    /**
+     * @dataProvider getRestrictionsValidations
+     */
+    public function testSimpleTypeWithValidations($xsRestrictions, $ymlValidations)
+    {
+        $xml = "
+             <xs:schema targetNamespace=\"http://www.example.com\" xmlns:xs=\"http://www.w3.org/2001/XMLSchema\">
+                <xs:element name=\"element-one\">
```

Comment:
this test is good, I will add another test, having `<xs:element name="element-one" maxOccours="unbounded">`